### ISSUE TITLE
Folders should show number of unarchived documents

### ISSUE THREAD

**Issue by nikhilmjn** [burtonator/polar-bookshelf#1237]
I want to know the number of unread (=unarchived) documents to manage my reading

**Describe the solution you'd like**
In the left pane, the each folder should state the number of unarchived documents inside it. It presently states the total number of documents.

<img width="302" alt="Screen Shot 2019-12-02 at 8 52 21 AM" src="https://user-images.githubusercontent.com/48635077/69928455-b8de6080-14e1-11ea-8906-159a3072fd56.png">

**Comment by burtonator** [comment]
Agreed. I'm going to work on that here shortly. I think if they show the unarchived numbers it will encourage users to keep these to zero. Plus I can have the folders being 'bold' showing that there are unread items similar to a news reader.

**Comment by thiswillbeyourgithub** [comment]
I think it's useful to have a way to know the total number of documents in each folder as well as knowing how many non archived there are.

Hence, to me the best way would be to show only the number of unarchived docs by default but when triggering on "archived" on the top right corner the numbers should update and show the total number.

Basically : the number of docs displayed should follow the same logic as the number of docs shown in the actual repository.

I am studying for my finals and it's annoying to not be able to see the numbers go down as a "todo" list.
